Makefile

```diff
@@ -104,6 +104,11 @@ image-apiserver:
 	docker build -f Containerfile.apiserver -t quay.io/apex/apiserver:$(TAG) .
 	docker tag quay.io/apex/apiserver:$(TAG) quay.io/apex/apiserver:latest
 
+.PHONY: image-apex
+image-apex:
+	docker build -f Containerfile.apex -t quay.io/apex/apex:$(TAG) .
+	docker tag quay.io/apex/apex:$(TAG) quay.io/apex/apex:latest
+
 .PHONY: images
 images: image-frontend image-apiserver ## Create container images
```

Comment:
Yeah,  I thought of that, but intentionally didn't add it, because we don't need this image locally for Kind setup. At this point of time we don't deploy agent in the Apex stack, so I think it's better to skip building it so we don't increase the local dev environment reload time.

Reply:
sounds good.  As another option, you could make sure it shows up in the `make` help text by adding ` ## Build the apex agent image` to the end of the line. I'd add it since it's meant to be called directly.

Reply:
fixed.